sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/kotlin/SourceRoots.kt

```diff
@@ -57,13 +58,28 @@ internal fun SqlDelightDatabase.sources(): List<Source> {
   ))
 }
 
-private fun KotlinMultiplatformExtension.sources(): List<Source> {
+private fun KotlinMultiplatformExtension.sources(project: Project): List<Source> {
   // TODO: Look at KotlinPlatformType when we get around to module dependencies and compatibility.
   // We'll probably want to include that in the source so we can tell which source to rely on
   // during dependency resolution.
 
   return targets
       .flatMap { target ->
+        if (target is KotlinAndroidTarget) {
+          project.extensions.findByType(BaseExtension::class.java)?.let {
```

Comment:
Can this be `getByType`? Can you have a Kotlin Android project without the Android plugin applied?

Reply:
nice, true